### PR TITLE
Convert the rushstack repo to a phased build.

### PR DESCRIPTION
**Pull request by iclanton**
This change converts the repo to build with two phases: `_phase:build` and `_phase:test`.

On my machine, `rush rebuild` goes from 5:45 without this change to 4:30 with this change.

### CODE REVIEW

File: common/config/rush/command-line.json [hunk 1]
```diff
@@ -11,6 +11,17 @@
    * "rush my-global-command --help".
    */
   "commands": [
+    {
+      "commandKind": "phased",
+      "name": "build",
+      "summary": "Phased build",
+      "description": "Ditto",
```
Comment (iclanton) [inline common/config/rush/command-line.json:18]: Update.

File: common/config/rush/rush-plugins.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
+/**
```
Comment (D4N14L) [inline common/config/rush/rush-plugins.json:1]: Is there a template for this file in the `rush init` files?

**Review by D4N14L — APPROVED** [review]

File: apps/rush/package.json [hunk 3]
```diff
@@ -24,8 +24,10 @@
   "engineStrict": true,
   "homepage": "https://rushjs.io",
   "scripts": {
-    "build": "heft test --clean",
-    "start": "node lib/start-dev-docs.js"
+    "build": "heft build --clean",
+    "start": "node lib/start-dev-docs.js",
```
[the remark this answers is not in the document]
Comment (iclanton) [reply apps/rush/package.json:28]: It just prints a note describing how to debug Rush.

File: common/config/rush/rush-plugins.json (hunk 2 above)
Comment (iclanton) [reply common/config/rush/rush-plugins.json:1]: Yeah https://github.com/microsoft/rushstack/blob/master/apps/rush-lib/assets/rush-init/common/config/rush/rush-plugins.json

**Pull request merged**